example/src/release_tests/gesturizedPressable/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactNode } from 'react';
+import { HitSlopExample } from './hitSlop';
+import { RippleExample } from './androidRipple';
+import { Text, View, StyleSheet } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { BG_COLOR } from './testingBase';
+import { FunctionalStyleExample } from './functionalStyles';
+import { DelayedPressExample } from './delayedPress';
+
+type TestingEntryProps = {
+  title: string;
+  platform?: string;
+  comment?: string;
+  children: ReactNode;
+};
+const TestingEntry = ({
+  children,
+  title,
+  platform,
+  comment,
+}: TestingEntryProps) => (
+  <View style={styles.container}>
+    <View style={styles.data}>
+      <Text style={styles.title}>{title}</Text>
+      {platform && <Text style={styles.code}>{platform}</Text>}
+    </View>
+    {comment && <Text style={styles.comment}>{comment}</Text>}
+    {children}
+    <View style={styles.separator} />
+  </View>
+);
+
+export default function Example() {
+  return (
+    <ScrollView style={styles.scrollable}>
+      <TestingEntry title="Hit Slop">
+        <HitSlopExample />
+      </TestingEntry>
+      <TestingEntry title="Ripple" platform="Android">
+        <RippleExample />
+      </TestingEntry>
+      <TestingEntry
+        title="Functional styling"
+        comment="lorem ipsum dolor sit amet">
+        <FunctionalStyleExample />
+      </TestingEntry>
+      <TestingEntry title="Delays on press callbacks">
+        <DelayedPressExample />
+      </TestingEntry>
+    </ScrollView>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: BG_COLOR,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  data: {
+    flex: 1,
+    alignSelf: 'flex-start',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    gap: 12,
+  },
+  title: {
+    fontSize: 28,
+    marginLeft: 12,
+    marginBottom: 5,
+    fontWeight: '400',
+  },
+  code: {
+    fontSize: 18,
+    fontWeight: '400',
+    padding: 5,
+    borderRadius: 5,
+    height: 34,
+    margin: 'auto',
+
+    color: '#37474f',
+    backgroundColor: '#bbc',
+    fontFamily: 'monospace',
+    fontVariant: ['tabular-nums'],
+  },
+  comment: {
+    alignSelf: 'flex-start',
+    marginLeft: 20,
+    marginTop: 0,
+    marginBottom: 5,
+    color: '#555',
+  },
+  scrollable: {},
+  separator: {
+    width: '90%',
+    // ios bug - different border styling doesn't work on just 1 side
```

Comment:
What do you mean?

Reply:
<img width="1498" alt="Screenshot 2024-06-24 at 10 21 52" src="https://github.com/software-mansion/react-native-gesture-handler/assets/74246391/96aa1b37-5010-4d7a-8b66-dd7575c62b25">

IOS doesn't allow for different styles for different borders.

Reply:
![image](https://github.com/software-mansion/react-native-gesture-handler/assets/74246391/9fd89198-4ef2-4538-a369-18e298efead0)
It's hard to explain

Reply:
![image](https://github.com/software-mansion/react-native-gesture-handler/assets/74246391/636648f8-c466-49f9-9722-76d2c36596b6)

Reply:
The key here is `borderStyle`, it's the only thing causing issue on IOS.

Reply:
And it works perfectly on web, here is how it is supposed to look :P

![image](https://github.com/software-mansion/react-native-gesture-handler/assets/74246391/17694bc5-fa1d-4270-85b9-e5292295e5fa)